### PR TITLE
Add `extra_gated_description ` to the gate docs

### PR DESCRIPTION
**Pull request by apolinario**
`extra_gated_description` was available but not documented

### CODE REVIEW

File: docs/hub/datasets-gated.md [hunk 1]
```diff
@@ -115,11 +115,12 @@ extra_gated_fields:
 ```
 
 
-In some cases, you might also want to modify the text in the gate heading and the text in the button. For those use cases, you can modify `extra_gated_heading` and `extra_gated_button_content` like this:
+In some cases, you might also want to modify the default text in the gate heading,content and the text in the button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```
Comment (apolinario) [inline docs/hub/datasets-gated.md:118]:
```suggestion
In some cases, you might also want to modify the default text in the gate heading, description and the text in the button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```

File: docs/hub/models-gated.md [hunk 2]
```diff
@@ -116,11 +116,12 @@ extra_gated_fields:
 ```
 
 
-In some cases, you might also want to modify the text in the gate heading and the text in the button. For those use cases, you can modify `extra_gated_heading` and `extra_gated_button_content` like this:
+In some cases, you might also want to modify the default text in the gate heading,content and the text in the button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```
Comment (apolinario) [inline docs/hub/models-gated.md:119]:
```suggestion
In some cases, you might also want to modify the default text in the gate heading, description and the text in the button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```

File: docs/hub/datasets-gated.md [hunk 3]
```diff
@@ -115,11 +115,12 @@ extra_gated_fields:
 ```
 
 
-In some cases, you might also want to modify the text in the gate heading and the text in the button. For those use cases, you can modify `extra_gated_heading` and `extra_gated_button_content` like this:
+In some cases, you might also want to modify the default text in the gate heading, description and the text in the button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```
Comment (pcuenca) [inline docs/hub/datasets-gated.md:118]:
```suggestion
In some cases, you might also want to modify the default text in the gate heading, description, and button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```

File: docs/hub/models-gated.md [hunk 4]
```diff
@@ -116,11 +116,12 @@ extra_gated_fields:
 ```
 
 
-In some cases, you might also want to modify the text in the gate heading and the text in the button. For those use cases, you can modify `extra_gated_heading` and `extra_gated_button_content` like this:
+In some cases, you might also want to modify the default text in the gate heading, description and the text in the button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```
Comment (pcuenca) [inline docs/hub/models-gated.md:119]:
```suggestion
In some cases, you might also want to modify the default text in the gate heading, description, and button. For those use cases, you can modify `extra_gated_heading`, `extra_gated_description` and `extra_gated_button_content` like this:
```

**Review by pcuenca — APPROVED** [review]
LGTM, thanks a lot! I'd wait to have confirmation from @coyotte508 or @osanseviero before merging :)

**Review by osanseviero — APPROVED** [review]
Thanks!

**Pull request merged**